yt_dlp/extractor/zdf.py

```diff
@@ -25,6 +24,11 @@ class ZDFBaseIE(InfoExtractor):
     _GEO_COUNTRIES = ['DE']
     _QUALITIES = ('auto', 'low', 'med', 'high', 'veryhigh', 'hd', 'fhd', 'uhd')
 
+    def _download_mediathekv2_document(self, document_id):
+        return self._download_json(
+            f'https://zdf-prod-futura.zdf.de/mediathekV2/document/{document_id}',
+            document_id)
+
```

Comment:
method name too long IMO
```suggestion
    def _download_v2_doc(self, document_id):
        return self._download_json(
            f'https://zdf-prod-futura.zdf.de/mediathekV2/document/{document_id}',
            document_id)
```